catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/CswEndpoint.java

```diff
@@ -813,6 +843,179 @@ public void unknownOperation() throws CswException {
         throw new CswException("No such operation", CswConstants.OPERATION_NOT_SUPPORTED, null);
     }
 
+    @Override
+    @DELETE
+    @Path("/subscription/{requestId}")
+    @Produces({MediaType.WILDCARD})
+    public Response deleteRecordsSubscription(@PathParam("requestId") String requestId)
```

Comment:
The interface and class are getting pretty long. Do you think we could split them up and have one interface/class for the root context and one for `/subscription`?